backend/rest/authRoutes.ts

```diff
@@ -167,4 +168,23 @@ authRouter.post(
   },
 );
 
+/* Reset password through a "Forgot Password" option */
+authRouter.post(
+  "/forgotPassword",
+  forgotPasswordRequestValidator,
+  async (req, res) => {
+    try {
+      try {
+        await userService.getUserByEmail(req.body.email);
+        await authService.resetPassword(req.body.email);
+        res.status(204).send();
+      } catch (error) {
```

Comment:
resetPassword:  @throws Error if unable to generate link or send email
getUserByEmail:  @throws Error if user retrieval fails

If an unrelated error occurred, would it be reported as EMAIL_NOT_FOUND ?